#!/usr/bin/env python
PACKAGE = 'r2_behavior'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
# General Settings for state
gen.add("disable_attention", bool_t, 0, "Disable attention control", False)
gen.add("disable_animations", bool_t, 0, "Disable animations", False)
gen.add("disable_blinking", bool_t, 0, "Disable blinking", False)
gen.add("disable_saccades", bool_t, 0, "Disable Sacades", False)
gen.add("disable_keepalive", bool_t, 0, "Disable keep alive head movement", False)
gen.add("chat_enabled", bool_t, 0, "Chatbot is enabled while in this state", True)
gen.add("performances_by_keyword", bool_t, 0, "Enable performances triggered by keyword", True)


exit(gen.generate(PACKAGE, "state", "State"))